network-mux/src/Network/Mux/Bearer/AttenuatedChannel.hs

```diff
@@ -11,7 +11,6 @@ module Network.Mux.Bearer.AttenuatedChannel
   , Attenuation (..)
   , newConnectedAttenuatedChannelPair
   , attenuationChannelAsMuxBearer
-  , CloseError (..)
```

Comment:
TODO: should be moved out of mux.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/3473